server/src/main/java/org/elasticsearch/cluster/routing/UnassignedInfo.java

```diff
@@ -430,22 +494,27 @@ public String toString() {
 
     @Override
     public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
-        builder.startObject("unassigned_info");
-        builder.field("reason", reason);
-        builder.field("at", DATE_TIME_FORMATTER.printer().print(unassignedTimeMillis));
+//        builder.startObject("unassigned_info");
```

Comment:
? are we handling this at a higher level now?

Reply:
yes, otherwise recreation is a hell